src/commands/hld/pipeline.ts

```diff
@@ -82,6 +82,15 @@ export const execute = async (
 ) => {
   try {
     populateValues(opts);
+    const hasPipelineFile = await repositoryHasFile("manifest-generation.yaml");
```

Comment:
Can you use the constants for the filename?

https://github.com/CatalystCode/spk/blob/master/src/lib/constants.ts#L17-L21